examples/novatel/rangecmp/range_decompressor_example.cpp

```diff
@@ -135,15 +135,19 @@ int main(int argc, char* argv[])
     stReadData.uiDataSize = sizeof(acFileStreamBuffer);
 
     unsigned char acFrameBuffer[MAX_ASCII_MESSAGE_LENGTH];
-    unsigned char* pucReadBuffer = acFrameBuffer;
+    unsigned char* pucFrameBuffer = acFrameBuffer;
     unsigned char acEncodeBuffer[MAX_ASCII_MESSAGE_LENGTH];
     unsigned char* pucEncodedMessageBuffer = acEncodeBuffer;
 
     InputFileStream clIFS(sInFilename.c_str());
     OutputFileStream clOFS(sInFilename.append(".DECOMPRESSED.").append(sEncodeFormat).c_str());
+    OutputFileStream clUnknownBytesOFS(sInFilename.append(".UNKNOWN").c_str());
     StreamReadStatus stReadStatus;
 
-    STATUS eStatus = STATUS::UNKNOWN;
+    auto eFramerStatus = STATUS::UNKNOWN;     // Framer Status
+    auto eDecompressStatus = STATUS::UNKNOWN; // Decompressor Status
+    auto eDecoderStatus = STATUS::UNKNOWN;    // Decoder Status
+    auto eEncoderStatus = STATUS::UNKNOWN;    // Encoder Status
```

Comment:
I dont see the point of making a description for a variable in a comment that is pretty much the same as the variable name